### PR TITLE
feat: first pass at XDK integration

### PR DESCRIPTION
**Pull request by bangarang**

### CODE REVIEW

File: src/xdk-simple-deploy.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import {
```
Comment (alexhollenbeck) [inline src/xdk-simple-deploy.ts:1]: Can we move this to src/examples/sheets?

File: src/xdk-simple-deploy.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+const TestSheet = new Sheet(
+  'TestSheet',
+  {
+    firstName: TextField('First Name'),
+    middleName: TextField('Middle'),
+    // linkedField: LinkedField({
+    //   label: 'Link to LinkedSheet',
+    //   sheet: LinkedSheet,
+    // }),
```
Comment (alexhollenbeck) [inline src/xdk-simple-deploy.ts:26]: Remove commented out code

Comment (alexhollenbeck) [reply src/xdk-simple-deploy.ts:26]: Or add linkedField back in if it's working

File: src/xdk-simple-deploy.ts (hunk 1 above)
Comment (bangarang) [reply src/xdk-simple-deploy.ts]: Moved to the `examples/workbook` folder.

File: src/xdk-simple-deploy.ts (hunk 2 above)
Comment (bangarang) [reply src/xdk-simple-deploy.ts]: Removed until we fully implement `LinkedField`

**Review by alexhollenbeck — APPROVED** [review]

**Pull request merged**